python/ray/_private/ray_constants.py

```diff
@@ -71,11 +71,10 @@ def env_set_by_user(key):
 
 ID_SIZE = 28
 
-# The default maximum number of bytes to allocate to the object store unless
-# overridden by the user.
-DEFAULT_OBJECT_STORE_MAX_MEMORY_BYTES = 200 * 10**9
```

Comment:
There is a comment for why having this cap: `# Cap memory to avoid memory waste and perf issues on large nodes`.

Can we leave this change out and just make the ratio configurable? Or also make this cap configurable.

Reply:
And that explanation isn't very useful, right? What's the waste? What are perf issues? No links, no context.

This cap of 200Gb is arbitrary and hard to explain for nodes that carry 1Tb+ memory available

Reply:
Sync'd offline. Restored the cap, made configurable instead.